proto/engine/v1/json_marshal_unmarshal.go

```diff
@@ -145,6 +146,8 @@ func (e *ExecutionPayload) MarshalJSON() ([]byte, error) {
 	timeStamp := hexutil.Uint64(e.Timestamp)
 	recipient := common.BytesToAddress(e.FeeRecipient)
 	logsBloom := hexutil.Bytes(e.LogsBloom)
+	fee := new(big.Int).SetBytes(bytesutil.ReverseByteOrder(e.Fees))
```

Comment:
So basically this will be a new JSON field of execution payloads that we receive via the engine API, right? This is gonna be hard unless we add the ability to skip fields for SSZ generation...

Reply:
Yeah. Or we don't use consensus protobuf object for engine API marshal/unmarshal. It's not clear to me why that's a must

Reply:
We shouldn't need to, only once we are preparing to put it inside a beacon block. The naming is weird though...perhaps we could have pb.ExecutionPayload which is used in consensus and ExecutionPayloadEngineResponse types

Reply:
> The naming is weird though

Geth uses `ExecutionData` which I like

Reply:
Great to see this feature being worked on!  To help co-ordinate amongst teams could you comment on the execution API issue/pull request on how to best format this new data field.  The current suggestion from @mkalinin is to create a new `engine_getPayloadV2` method.

https://github.com/ethereum/execution-apis/pull/314
https://github.com/ethereum/execution-apis/issues/307